packages/date-time-picker/test/value-commit.test.js

```diff
@@ -0,0 +1,376 @@
+        await clearInput(picker);
+        await sendMouseToElement({ type: 'click', element: otherPicker });
+        await expectNoValueCommit();
+      });
+
+      it('should commit on date-time-picker outside click', async () => {
+        await clearInput(picker);
+        outsideClick();
+        await expectValueCommit();
+      });
+    });
+  });
+  describe('initially invalid value', () => {
+    beforeEach(async () => {
+      dateTimePicker.min = '2011-01-01T12:00';
+      await enterInput(datePicker, '2/2/2002');
+      await sendKeys({ press: 'Enter' });
+      await enterParsableInput(timePicker);
+      outsideClick();
+      await nextRender();
+      resetSpyHistories();
+    });
+
+    describeForEachPicker('value partially cleared (= incomplete value)', (pickerType) => {
+      let picker, otherPicker;
+
+      beforeEach(() => {
+        [picker, otherPicker] = getPicker(pickerType);
+      });
+
+      it('should commit on Enter', async () => {
+        await clearInput(picker);
+        await sendKeys({ press: 'Enter' });
+        await expectValueCommit();
+      });
+
+      it('should commit on other picker click', async () => {
+        await clearInput(picker);
+        await sendMouseToElement({ type: 'click', element: otherPicker });
+        await expectValueCommit();
+      });
+
+      it('should commit on date-time-picker outside click', async () => {
+        await clearInput(picker);
+        outsideClick();
+        await expectValueCommit();
+      });
+    });
+  });
+
+  describe('defined range', () => {
+    beforeEach(async () => {
+      dateTimePicker.min = '1980-02-02T02:00';
+      dateTimePicker.max = '1990-02-02T02:00';
+      await nextRender();
+      resetSpyHistories();
+    });
+
+    it('should commit on date-picker Enter when value is changed to a date outside the set range', async () => {
+      await enterParsableInput(timePicker);
+      await enterInput(datePicker, '2/2/2002');
+      resetSpyHistories();
+      await sendKeys({ press: 'Enter' });
+      await expectValueCommit();
+    });
+
+    it('should commit on date-picker outside click when value is changed to a date outside the set range', async () => {
+      await enterParsableInput(timePicker);
+      await enterInput(datePicker, '2/2/2002');
+      resetSpyHistories();
+      outsideClick();
+      await expectValueCommit();
+    });
+
+    it('should commit as unparsable on date-picker outside click when value is changed to a date inside the set range while time-picker is empty', async () => {
+      await enterInput(datePicker, '2/2/1985');
+      resetSpyHistories();
+      outsideClick();
+      await expectUnparsableValueCommit();
+    });
+
+    it('should commit on date-picker outside click when value is changed to a date inside the set range while time-picker is filled', async () => {
+      await enterParsableInput(timePicker);
+      await enterInput(datePicker, '2/2/1985');
+      resetSpyHistories();
```

Comment:
Just entering input shouldn't result in a commit. The same applies to other tests in this suite.

```suggestion
```

Reply:
Removed the reset from the tests.